R/chrome.R

```diff
@@ -246,17 +272,25 @@ get_entrypoint = function(debug_port) {
 
 print_page = function(
   ws, url, output, wait, verbose, token, format,
-  options = list(), selector, box_model, scale
+  options = list(), selector, box_model, scale, resolve, reject
 ) {
 
   ws$onMessage(function(binary, text) {
-    if (!is.null(token$error)) return(ws$close())
+    if (!is.null(token$error)) {
+      res = ws$close()
+      reject(token$error)
+      return(res)
```

Comment:
I know you want to be careful here. I think the code `return(ws$close())` was written by me, and I didn't mean the returned value was meaningful. I just wanted to `return()` early. I guess here you don't really need to assign `res = ws$close()` and return it. Same thing on line 292.

Reply:
done